docs/core/compatibility/toc.yml

```diff
@@ -63,6 +63,8 @@ items:
           href: aspnet-core/6.0/clientcertificate-doesnt-trigger-renegotiation.md
         - name: EndpointName metadata not set automatically
           href: aspnet-core/6.0/endpointname-metadata.md
+        - name: "Identity: Default Bootstrap version of UI changed"
+          href: aspnet-core/6.0/identity-bootstrap4-to-5.md
```

Comment:
This should also be added to https://github.com/dotnet/docs/blob/main/docs/core/compatibility/toc.yml.

Reply:
Isn't that this file?